projects/storefrontlib/src/cms-components/product/carousel/product-references/product-references.component.spec.ts

```diff
@@ -103,6 +103,8 @@ class MockProductReferenceService {
   get(_code: string): Observable<ProductReference[]> {
     return of([mockProductReferences[0], mockProductReferences[1]]);
   }
+
+  cleanReferences(): void {}
 }
```

Comment:
It's better add one more test, that when `ngOnInit` called, expect `cleanReferences` has been called.